### PR TITLE
modify the README.md of fusedemo

### PR DESCRIPTION
**Pull request by 13186379707**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by 13186379707** [comment]
modify the README.md of fusedemo

**Review by helight — APPROVED** [review]
LGTM

File: eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md [hunk 1]
```diff
@@ -1,18 +1,33 @@
...
+//编译linfuse库
+ninja
+sudo ninja install
 ```
 
 ## 运行(挂载)：
 
 ```
+cd ./example/
 mkdir yourdir
```
Comment (helight) [inline eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md:31]: 这部分好像不对，example 目录是那个？

File: eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
 ./fusedemo yourdir
 ```
```
Comment (helight) [inline eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md:33]: fusedemo 需要一个 makefile 文件来组织编译过程

File: eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md [hunk 3]
```diff
@@ -24,6 +39,18 @@ mkdir yourdir
 df -T yourdir
```
Comment (helight) [inline eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md:39]: 测试也可以是 makefile 中管理

make test 来运行

File: eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md (hunk 1 above)
Comment (13186379707) [reply eBPF_Supermarket/Filesystem_Subsystem/fast_fuse/fusedemo/README.md:31]: 经过meson与ninja的编译之后，build目录中会生成example目录，其中会生成fusedemo文件系统的可执行文件。

**Review by chenamy2017 — APPROVED** [review]

**Pull request merged**